packages/content/src/Infrastructure/Sulu/Preview/ContentObjectProvider.php

```diff
@@ -90,81 +90,57 @@ public function getObject($id, $locale)
                 ->getQuery()
                 ->getSingleResult();
         } catch (NoResultException $exception) {
-            return null;
+            return [];
         }
 
-        return $this->resolveContent($contentRichEntity, $locale);
+        $object = $this->resolveContent($contentRichEntity, $locale);
+
+        if (!$object instanceof RoutableInterface) {
+            throw new \RuntimeException('Object without RoutableInterface need create there own PreviewDefaultsProvider');
+        }
+
+        return [
+            'object' => $object,
+            '_controller' => ContentController::class . '::indexAction', // TODO call RouteDefaultsProvider
+            'view' => 'base', // TODO should provided by RouteDefaultsProvider
+        ];
```

Comment:
This changes are just a test we need implement this in: https://github.com/sulu/sulu/pull/7749 where the new routing will be implemented